tools/swift-api-digester/ModuleAnalyzerNodes.cpp

```diff
@@ -423,6 +429,11 @@ bool SDKNodeDeclType::isConformingTo(KnownProtocolKind Kind) const {
   }
 }
 
+bool SDKNodeDeclType::hasFixedLayout() const {
```

Comment:
This will still return false for non-resilient modules, where all declarations are fixed-layout/frozen. Instead, use `NominalTypeDecl::isResilient()`.